keyboards/system76/launch_1/usb_mux.c

```diff
@@ -0,0 +1,478 @@
+/*
```

Comment:
This should be pulled out to a separate PR, and added to `/drivers/`, instead.

Reply:
Would rather that didnt happen, and was done when a 2nd keyboard needs the same functionallity.